next/Dockerfile.prod

```diff
@@ -20,6 +20,12 @@ RUN \
 
 # Rebuild the source code only when needed
 FROM base AS builder
+
+# defaults for use during the container build phase on CI/CD
+ARG IMAGES_PROTOCOL=https
+ARG IMAGES_HOSTNAME=**
+ARG IMAGES_PORT=433
```

Comment:
I want to revisit this change before considering this PR ready.

Setting the hostname to `**` is most likely a security issue, but we do want to allow several hostnames so we can use the same CI built image without having it be rebuilt for each hostname. The patterm `**.rabe.ch` might fulfill this, or we add a regex that is more specific.

Potentially we could go even further and also add `pathname` to narrow down the allowed image URLs.

Note to self: the docs for the feature are here: https://nextjs.org/docs/pages/api-reference/components/image#remotepatterns (and the env vars are defined in next.config.js).

Reply:
I changed it to `*.rabe.ch` during build-time in the CI pipeline, but that pattern still allows sites like `share.rabe.ch`.

Preferably we would like for it to not be possible to optimize content from ownCloud via nextjs.